sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TransmitFromStorageHandler.cs

```diff
@@ -61,8 +61,11 @@ internal void TransmitFromStorage(object? sender, ElapsedEventArgs? e)
 
                             // In case if the delete fails, there is a possibility
                             // that the current batch will be transmitted more than once resulting in duplicates.
-                            // TODO: TryDelete returns a boolean, should we log when this occurs?
-                            blob.TryDelete();
+                            var deleteSucceeded = blob.TryDelete();
+                            if (!deleteSucceeded)
```

Comment:
@vishweshbankwar Should we consider logging this message. Whoever analyzes might see this an issue. No strong opinion from my side.

Reply:
if users complaints about duplicate telemetry, this is one possible scenario we would like to check. Ideally, we should not run in to this situation.